crates/bin/pd/src/info/oblivious.rs

```diff
@@ -269,7 +269,7 @@ impl ObliviousQueryService for Info {
                         .expect("compact block for in-range height must be present");
 
                     let send_op = async { tx_blocks.send(Ok(block.into())).await };
-                    match tokio::time::timeout(tokio::time::Duration::from_secs(1), send_op).await {
+                    match tokio::time::timeout(tokio::time::Duration::from_secs(2), send_op).await {
```

Comment:
I think it would be better to revert the timeouts entirely, so we can move on to other more pressing tasks and return to optimizing sync performance (which is not consensus critical) at a later point in time.

Reply:
I tried removing all timeouts, but got a little tripped up sorting through the various recent commits. @erwanor can you take a look? I'd like to get another patch release out, to unblock the Zpoken folks.

Reply:
Sent! thanks for your help with this